benchmarker_utils.f90

```diff
@@ -40,6 +40,36 @@ subroutine assert_real_2d(a, b, test_name, rtol_opt)
 
   end subroutine assert_real_2d
 
+  subroutine assert_real(a, b, test_name, rtol_opt)
+
+    implicit none
+
+    character(len=*) :: test_name
+    real, intent(in) :: a, b
+    real, optional :: rtol_opt
+    real :: relative_error, rtol
+
+    character(len=15) :: pass, fail
+
+    fail = char(27)//'[31m'//'FAILED'//char(27)//'[0m'
```

Comment:
Not necessary for this PR but we could think about ways to remove code duplication in the assertion statements.

Reply:
Thanks, yes I agree. I've added it as a to-do for the general tidy up in #24